src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,31 @@
-const fizzBuzz = (number) => {};
+// const fizzBuzz = (number) => {
+//   if (number % 5 === 0 && number % 3 === 0) {
+//     return "FizzBuzz";
+//   }
+//   if (number % 5 === 0) {
+//     return "Buzz";
+//   }
+//   if (number % 3 === 0) {
+//     return "Fizz";
+//   }
+//   return number;
+// };
 
+const fizzBuzz = (number) => {
```

Comment:
Nice! I like that you have taken the time to create the `isDivisible` variables to make the code easier to understand.